server/apps/archive/common.py

```diff
@@ -41,6 +41,13 @@
 SEQUENCE = 'sequence'
 PUBLISH_STATES = ['published', 'killed', 'corrected', 'scheduled']
 CUSTOM_HATEOAS = {'self': {'title': 'Archive', 'href': '/archive/{_id}'}}
+ITEM_OPERATION = 'operation'
+ITEM_CREATE = 'create'
+ITEM_UPDATE = 'update'
+ITEM_RESTORE = 'restore'
+ITEM_DUPLICATE = 'duplicate'
+ITEM_DESCHEDULE = 'deschedule'
+item_operations = [ITEM_CREATE, ITEM_UPDATE, ITEM_RESTORE, ITEM_DUPLICATE, ITEM_DESCHEDULE]
 
```

Comment:
As suggested by @plamut in one of the earlier PR's it would be good if we can create a `namedtuple()` for this instead of list